docs/checkstyle/check-docs.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -ue
+
+ret=0
+cd docs/guides
+
+for docs in admin developer; do
+  cd $docs
+  echo "Markdown doc $docs build log:"
+  mkdocs build &> mkdocs.log
+  cat mkdocs.log
+  if grep \
+       -e 'WARNING.*Documentation file' \
+       -e 'pages exist in the docs directory' \
+       -e 'is not found in the documentation files' \
+        mkdocs.log;
```

Comment:
Why these specific errors only? Why not also fail on "ERROR"? Or just check the exit code of `mkdocs`?

Reply:
With the modification to this test, it does exactly that contrary to what happened with the `| tee ...` before.
This means, the test now fails if `mkdocs` fails.
These are just additional things to check for.